pyproject.toml

```
@@ -32,40 +32,44 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
 ]
-
-packages = [
-    { include = "linkml_runtime" }
+include = ["linkml_runtime"]
+requires-python = ">=3.8"
+dynamic = ["version", "readme"]
```

```suggestion
dynamic = ["version"]
```

Readme isn't dynamic, right? Or is this some poetry thing